public/js/office.io.web.js

```diff
@@ -0,0 +1,61 @@
+function OfficeIo(config){
```

Comment:
I would rename it to `OfficeIO`, or to something that is not related to IO but with events, for example, `OfficeEvents` because of this module is responsible to emit and handle events.

Reply:
good Idea :)